pandas/core/groupby/groupby.py

```diff
@@ -1743,7 +1743,8 @@ def cummin(self, axis=0, **kwargs):
         if axis != 0:
             return self.apply(lambda x: np.minimum.accumulate(x, axis))
 
-        return self._cython_transform('cummin', numeric_only=False)
+        return self._cython_transform('cummin',
```

Comment:
is there a test for these?

Reply:
I think so. I’ll have to get back to you with the specific test case.

Reply:
I think so. I’ll have to get back to you with the specific test case.

Reply:
ok was happy to have you do it here. ping on green and can merge this PR.